src/common/browser-adapters/browser-adapter-event-manager.ts

```diff
@@ -26,10 +25,13 @@ const FOUR_MINUTES = 2 * TWO_MINUTES;
 export class BrowserAdapterEventManager {
```

Comment:
It would maybe be helpful to give this class a comment explaining the problem it's solving; I worry that it would be easy for a maintainer without context to remove a subtly important timeout